src/System.Device.Gpio/Interop/Unix/libgpiod/V2/LastErr.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// Disable these StyleCop rules for this file, as we are using native names here.
+#pragma warning disable SA1300 // Element should begin with upper-case letter
+
+using System.Runtime.InteropServices;
+
+namespace System.Device.Gpio.Interop.Unix.libgpiod.V2;
+
+internal static class LastErr
```

Comment:
NIT: Can't this be also used for V1?

Reply:
Er... yes. Basically anywhere where` GetLastWin32Error()` is used. I thought it would be good that when migrating to >=NET7, this feature is also used.

Reply:
Smth to be done about this?